Kickstarter-iOS/Views/Cells/SettingsAccountWarningCell.swift

```diff
@@ -14,13 +14,6 @@ final class SettingsAccountWarningCell: UITableViewCell, ValueCell, NibLoading {
 
     _ = self.warningIconImage
       |> \.isHidden .~ shouldHideAlertIcon
-
-    if !shouldHideAlertIcon {
```

Comment:
Was this done by an accident? It effectively reverts changes done in https://github.com/kickstarter/ios-oss/pull/535/files 😄

Reply:
Had a merge conflict, going to correct that now